readme_renderer/clean.py

```diff
@@ -54,21 +56,75 @@
     "width", "height",
 ]
 
+ALLOWED_CLASSES = {
+    "img": ["align-left", "align-center", "align-right"],
+    "span": [c for c in pygments.token.STANDARD_TYPES.values() if c],
```

Comment:
Two things:

* I'm not sure if there is any use of a class inside of a span that isn't there to enable pygments, or if the list of pygments classes is all we need to worry about here.
* @theacodes has set the ``<code>`` element to allow classes in the PR that added syntax highlighting, but I can't determine what classes are valid for the ``<code>`` element. Trying different combinations of things doesn't seem to actually generate code that emits a class on the ``<code>`` element. If we can't figure out how to make a README emit a ``<code>`` element with css classes, we should probably emit ``class`` from ``<code>``.